typha/fv-tests/server_test.go

```diff
@@ -901,7 +905,9 @@ var _ = Describe("With an in-process Server with long ping interval", func() {
 			},
 		)
 		err := client.Start(clientCxt)
-		go recorder.Loop(clientCxt)
+		recorderCtx, recorderCancel := context.WithCancel(context.Background())
+		defer recorderCancel()
+		go recorder.Loop(recorderCtx)
```

Comment:
wouldn't it make more sense to start it after checking the Start() error `Expect(err).NotTo(HaveOccurred())` may not make any difference with the defer though.

Reply:
Oh yeah, that would make sense (was just doing a handle-turn find/replace)

Reply:
Gonna leave this PR as is because I've already backported to enterprise; will put up a follow-on PR